src/_data/sites/en.yml

```
@@ -491,21 +498,25 @@ donate_page:
         Donate <span class="visually-hidden">on GitHub</span>
     items:
       - name: Platinum Sponsor
```

```suggestion
      - name: Diamond Sponsor
```
Diamond - Sponsors will contribute over $5,000

---

@nzakas  can you check this once?  the name should be Diamond Sponsor, right?

---

merged this change as its pretty simple